lib/puppet/provider/rhsm_reconfigure_script/rhsm_reconfigure_script.rb

```diff
@@ -125,7 +125,6 @@ def script_template
 
       # also add the katello ca cert to the system wide ca cert store
       if [ -d $CA_TRUST_ANCHORS ]; then
-        update-ca-trust enable
         cp $CERT_DIR/$KATELLO_SERVER_CA_CERT $CA_TRUST_ANCHORS
         update-ca-trust
```

Comment:
Should this be changed to explicitly extract? Similar to https://github.com/voxpupuli/puppet-trusted_ca/pull/69#discussion_r1724424510
```suggestion
        update-ca-trust extract
```

Reply:
I'd do that, yeah